src/client.c

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2009-2015 Keldon Jones
  *
  * Source file modified by B. Nordli, August 2015.
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
If connected to old server (check "server_version"), Xeno Invasion radio button must be disabled. Do this in the loop around 2720.